.github/workflows/ci.yml

```
@@ -0,0 +1,90 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  qa:
+    name: QA
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toxenv:
+          - checkqa
+          - readme
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+      - name: Set PY
+        run: echo "::set-env name=PY::$(python -VV | sha256sum | cut -d' ' -f1)"
+      - uses: actions/cache@v1
+        with:
+          path: ~/.cache/pre-commit
+          key: pre-commit|${{ env.PY }}|${{ hashFiles('.pre-commit-config.yaml') }}
+      - name: Install tox
+        run: pip install tox
+      - name: Test ${{ matrix.toxenv }}
+        run: tox -e ${{ matrix.toxenv }}
```

Better set it via env var https://github.com/jazzband/pip-tools/pull/1012/files#r419001428:
```suggestion
        run: tox
```

---

I intentionally didn't use env to be consistent with `cpython|pypy` jobs.

---

I think it's cleaner to use env everywhere. Besides, you still need to declare other env vars.

---

> Besides, you still need to declare other env vars.

Which ones?

---

The ones that tox understands: https://github.com/jazzband/pip-tools/pull/1012#discussion_r419001428.

Plus, it's also the best practice to have a separate step with `tox --notest` in CI.

---

> I think it's cleaner to use env everywhere. Besides, you still need to declare other env vars.

Could you suggest env solution for `cpython` job?

---

sure

---

Thanks! Looks good.  Addressed in 988dfbc.